doc/src/tutorials/arch/heterogeneous_tiles.rst

```diff
@@ -0,0 +1,158 @@
+.. _heterogeneous_tiles_tutorial:
+
+Heterogeneous tiles tutorial
+============================
+
+This tutorial aims at providing information to the user on how to model sub tiles to enable ``heterogeneous tiles`` in VPR.
+
+To correctly model an architecture, ``each physical tile`` requires at least one sub tile definition. This represents a default
+homogeneous architecture, composed of one or many capacity instances of the sub tile within the physical tile.
+
+To enhance the expressivity of VPR architecture, additional sub tiles can be inserted alongside with the default sub tile.
+This enables the definition of so called ``heterogeneous tiles``.
+
+An ``heterogeneous tile`` is a tile that includes a two or more site types that may differ in the following aspects:
+
+- ``Fc definition``
+- ``pinlocations``
+- ``IO ports definition``
+
+With this new capability, the device grid of a given architecture does include a new ``depth`` coordinate that identifies
+the type of sub tile used and its actual location, in case the capacity is greater than 1.
+
+A visual representation of the new VPR grid is the following:
+
+.. image:: sub_tiles_grid.png
+
+Heterogeneous tiles examples
+----------------------------
+
+1. Different pin locations among sub tiles.
+
+   The Xilinx Series 7 Clock tile is composed of 16 BUFGCTRL sites. Even though they are equivalent regarding the ports and Fc
```

Comment:
Maybe  we can add a link to the Xilinx Series 7 clock tile documentation, so that people can look up what the hardware this is modelling actually looks like? Perhaps we could also have a diagram illustrating what this is modelling?

Otherwise this example will be hard to follow for those who don't know the low-level details of 7 series clocking tiles.